config/packages/yadm.yaml

```diff
@@ -0,0 +1,41 @@
+parameters:
+    mongo_database: '%env(PAYUM_MONGO_DATABASE)%'
+    mongodb.uri: "%env(PAYUM_MONGO_URI)%"
+
+services:
+
+    # default configuration for services in *this* file
+    _defaults:
+        autowire: true      # Automatically injects dependencies in your services.
+        autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
+        public: false       # Allows optimizing the container by removing unused services; this also means
+                            # fetching services directly from the container via $container->get() won't work.
+                            # The best practice is to be explicit about your dependencies anyway.
+
+
+yadm:
+    mongo_uri: "%mongodb.uri%"
+    models:
+        gateway_config:
+            class: 'App\Model\GatewayConfig'
+            collection: 'gateway_configs'
+            database: '%mongo_database%'
+            hydrator_class: 'App\Yadm\Hydrator\GatewayConfigHydrator'
+            storage_class: 'App\Storage\GatewayConfigStorage'
+            pessimistic_lock: true
```

Comment:
We do not need a pessimistic_lock for this and other models

Reply:
Roger roger.